templates/ingress/ingress.tmpl.yaml

```diff
@@ -0,0 +1,287 @@
+---
```

Comment:
I'm really confused about this change. Can you explain what its doing? Also - probably don't want to hard-code the k8s namespace.

Reply:
This was to pin the ingress version rather than dynamically pulling the latest `ingress.yaml` on startup. This requirement is probably more relevant for offline installations or reproducible environments.

If you want to handle the ingress installation manually, run with `SKIP_INGRESS=true` to skip applying this file.